org.eclipse.paho.android.service/src/main/java/org/eclipse/paho/android/service/HandlerPingSender.java

```diff
@@ -0,0 +1,110 @@
+package org.eclipse.paho.android.service;
```

Comment:
Normally I'd say that this code could probably have been done in Kotlin, but I know that this project doesn't have Kotlin support enabled.